sr_logging_common/logging/bag_rotate.py

```diff
@@ -40,29 +28,20 @@ def __init__(self, path, desired_bag_number, prefix):
         self.run()
 
     def remove_old_bags(self):
-        bags_to_remove = self.get_suffixed_bags(".bag")
-        bags_to_remove.extend(self.get_suffixed_bags(".bag.orig.active"))
+        bags_to_remove = self.get_file_paths_by_key(".bag")
```

Comment:
We should rename `bags_to_remove` to `bags` or `existing_bags`.

Reply:
changed in [b2f2f19](https://github.com/shadow-robot/common_resources/pull/257/commits/b2f2f194c91ac46ccf033b0535821f4dc47e5f46)